examples/results/forces.py

```diff
@@ -0,0 +1,60 @@
+import flow360 as fl
+from flow360.examples import OM6wing
+
+OM6wing.get_files()
+
+# submit mesh
+volume_mesh = fl.VolumeMesh.from_file(OM6wing.mesh_filename, name="OM6wing-mesh")
+volume_mesh = volume_mesh.submit()
+
+# submit case using json file
+params = fl.Flow360Params(OM6wing.case_json)
+case = volume_mesh.create_case("OM6wing-example", params)
+case = case.submit()
+
+# wait until the case finishes execution
+case.wait()
```

Comment:
Can we have automatic wait when results are requested?

Reply:
Actually, you can get partial results before case is finished